build/charts/antrea/crds/packetcapture.yaml

```diff
@@ -109,7 +109,8 @@ spec:
                       enum: [IPv4]
                       default: IPv4
                     protocol:
-                      x-kubernetes-int-or-string: true
+                      type: string
+                      enum: [TCP,UDP,ICMP]
```

Comment:
My rational was to avoid introducing an uncommon go module to convert protocol string to number. The proposal that supports arbitrary protocol number and a limited set of protocol strings work for me too.

Reply:
updated. use `x-kubernetes-int-or-string: true` for protocol. add validation check before sync CR.